### DISCUSSION TITLE
Can't upload after migrating Minio to local storage

### DISCUSSION THREAD

**Discussion by oakbrad** [outline/outline#6528] [Self Hosting]
I am trying to migrate away from using Minio, and have followed the [instructions in the documentation](https://docs.getoutline.com/s/hosting/doc/file-storage-N4M0T6Ypu7#h-file-system-folder).

Previously I had Minio running in Docker and had it store the data with a bind mount to my local file system. I shut Minio down, and then bound that same directory to Outline as `/var/lib/outline/data`

Works:
* Images in my documents display correctly

Does Not Work:
* Uploading new images or attachments
* Downloading large attachments (ie. a 5MB PDF works fine but not a 80MB one)

To rule out permissions issues I set the data directory to 777 but I still get "Upload Failed".

Unsure how to troubleshoot this further

**Comment by tommoor** [comment]
@oakbrad what is the error in the browser console and/or in the server logs?

**Comment by oakbrad** [comment]
Fixed with successful upgrade to 0.74.0
